src/Styleguide/Components/CountrySelect.tsx

```diff
@@ -0,0 +1,258 @@
+import React from "react"
+import { PositionProps, SpaceProps } from "styled-system"
+import { LargeSelect } from "Styleguide/Elements/Select"
+
+export interface CountrySelectProps extends PositionProps, SpaceProps {
+  selected?: string
+  onSelect?: (value) => void
+}
+
+export const CountrySelect = (props: CountrySelectProps) => {
+  return <LargeSelect {...props} options={CountrySelectOptions} />
+}
+
+const CountrySelectOptions = [
```

Comment:
maybe `countrySelectOptions` or `COUNTRY_SELECT_OPTIONS`? TitleCase is typically reserved for classes / components and static data typically follows a conventional const format.

Reply:
fixed